src/rapids_pre_commit_hooks/copyright.py

```diff
@@ -114,6 +114,10 @@ class ConflictingFilesWarning(RuntimeWarning):
     pass
 
 
+def force_spdx(args: argparse.Namespace) -> bool:
+    return args.force_spdx or bool(int(os.getenv("FORCE_SPDX", default="0")))
```

Comment:
Proposing a tightly-scoped name here.
```suggestion
    return args.force_spdx or bool(int(os.getenv("RAPIDS_COPYRIGHT_FORCE_SPDX", default="0")))
```